src/components/Header/Header.tsx

```diff
@@ -31,9 +32,15 @@ export default function Header() {
             ☕ Coffee
           </Typography>
 
-          <Button variant="outlined" color="default" startIcon={<ExitToApp />}>
-            LOGOUT
-          </Button>
+          <Link to="/login" className="logout-button">
+            <Button

```

Comment:
please, add `aria-label` in order to clarify `Button` purpose for anyone who cannot access the visual indication.

Reply:
we will not support accessability